src/main/java/frc/robot/Robot.java

```diff
@@ -279,28 +305,47 @@ public void teleopSetup() {
     elevator.start();
     shooter.start();
     ballSubsystem.start();
-  }
 
+
+  }
+  TalonSRX turretMotor = new TalonSRX(RobotMap.TURRET_PORT);
+  TalonFX shooterMotor = new TalonFX(RobotMap.SHOOTER_PORT);
   /** This function is called periodically during operator control. */
   @Override
   public void teleopRoutine() {
-      double forward = this.driverPad.getValue(ControllerBindings.RIGHT_STICK, ControllerBindings.Y_AXIS);
-      double turn = this.driverPad.getValue(ControllerBindings.LEFT_STICK, ControllerBindings.X_AXIS);
+      double forward, turn;
+
+
+      forward = -this.operatorPad.getValue(ControllerBindings.LEFT_STICK, ControllerBindings.Y_AXIS);
```

Comment:
The sneaky little `-` in front of the read value is easy to miss. Better to be explicit and use the `InvertedJoystickMode` like we have at the top of `teleopSetup`.